packages/qwik/src/core/state/common.ts

```diff
@@ -490,11 +499,15 @@ export class LocalSubscriptionManager {
         } else {
           const signal = sub[SubscriptionProp.SIGNAL];
           if (type == SubscriptionType.PROP_IMMUTABLE || type == SubscriptionType.PROP_MUTABLE) {
-            scheduler.$scheduleNodeProp$(
-              host as fixMeAny,
-              sub[SubscriptionProp.ELEMENT] as fixMeAny,
-              sub[SubscriptionProp.ELEMENT_PROP],
-              signal
+            const target = sub[SubscriptionProp.ELEMENT] as fixMeAny as VirtualVNode;
+            const propKey = sub[SubscriptionProp.ELEMENT_PROP];
+            updateNodeProp(
```

Comment:
I would expect a call to `container.scheduleRender()` so that the journal gets flushed.